commands/Moderation/unmute.js

```diff
@@ -0,0 +1,28 @@
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			permissionLevel: 6,
+			requiredPermissions: ['MANAGE_ROLES'],
+			runIn: ['text'],
+
+			description: 'Unmutes a mentioned user.',
+			usage: '<member:user> [reason:string] [...]',
```

Comment:
member should use the member argument and not user because it does the same thing you're doing.

Reply:
This is due to already established conventions in ban and softban